### PR TITLE
Hello branch

### PR DESCRIPTION
**Pull request by Arcticleech**
Please be kind

### CODE REVIEW

File: Hello [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+My name is sean.
+
+I like coffee,
+
+even if it is cold.
+
+:)
+
+ITS MUCH BETTER COLD
```
Comment (kiemnguyen189) [inline Hello]:
```suggestion
ITS MUCH BETTER HOT
```

**Review by kiemnguyen189 — CHANGES_REQUESTED** [review]
Much better now.

**Review by kiemnguyen189 — COMMENTED** [review]
there we go

**Review by kiemnguyen189 — APPROVED** [review]
I like it hot now thank you

**Comment by Arcticleech** [comment]
This was all about coffee

**Pull request merged**